store/postgres_migration.go

```diff
@@ -27,7 +28,11 @@ func psqlGenerateMigration(provider config.StoreProviderType, tenant string, ch
 			case fieldType:
 				m = append(m, "ALTER TABLE IF EXISTS "+psqlAbsTableName(tenant, tableName)+" DROP COLUMN IF EXISTS "+change.TableInfo.ElementName)
```

Comment:
❓ Is this... a fragment of (SQL injection) attack surface? 
❓ Do `tenant` and `tableName` come already sanitized?

Reply:
There will be naming conventions for these. `tenant` and `tableName` will both be something like `[A-Za-b0-9-_]`

Reply:
The validation of names has yet to be added...